docs/management/api/_exceptions-api-reusable-content.asciidoc

```diff
@@ -83,7 +83,7 @@ created when it is not provided.
 |`meta` |Object |Placeholder for metadata about the exception item. |No
 |`name` |String |The {endpoint-artifact-name-single} name. |Yes
 |`namespace_type` |String a|Must be set to `agnostic` for endpoint artifacts. |Yes
-|`os_types` |xref:exception-{endpoint-artifact-api-doc-name}-item-os-types[os_type][] |Array of xref:exception-{endpoint-artifact-api-doc-name}-item-os-types[os_type] values. | Yes
+|`os_types` |xref:exception-{endpoint-artifact-api-doc-name}-item-os-types[os_type][] |Use this field to specify the xref:exception-{endpoint-artifact-api-doc-name}-item-os-types[operating system]. | Yes
```

Comment:
Can we say here:

```
Use this field to specify the xref:exception-{endpoint-artifact-api-doc-name}-item-os-types[operating system].  You can use only one value.
```

Just to make it clear that you can only use one OS value

Reply:
```suggestion
|`os_types` |xref:exception-{endpoint-artifact-api-doc-name}-item-os-types[os_type][] |Use this field to specify the xref:exception-{endpoint-artifact-api-doc-name}-item-os-types[operating system]. Only enter one value.  | Yes
```